pom.xml

```diff
@@ -67,6 +67,14 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.mule.module</groupId>
+            <artifactId>mule-java-module</artifactId>
+            <version>1.3.0-SNAPSHOT</version>
```

Comment:
Is it necessary to use the SNAPSHOT version? It would be better to point to a released version.

Reply:
Not necessary, changing to 1.2.1